Gemfile

```diff
@@ -16,7 +16,7 @@ gem 'jbuilder', '~> 1.2'
 gem 'piwik_analytics', '~> 1.0.2'
 
 gem 'geocoder', '~> 1.2.1'
-gem 'activeadmin', git: 'https://github.com/gregbell/active_admin.git'
+gem 'activeadmin', '1.0.0.pre2'
```

Comment:
This isn't strictly necessary, but there's a quirk in that repository that causes fetches to fail when the local Git settings enforce strict integrity checking: https://github.com/activeadmin/activeadmin/issues/4361